http-client/src/main/java/io/micronaut/http/client/netty/DefaultNettyHttpClientRegistry.java

```diff
@@ -125,6 +126,7 @@
      * @param eventLoopGroupFactory           The event loop group factory
      * @param beanContext                     The bean context
      * @param invocationInstrumenterFactories The invocation instrumenter factories
+     * @param jsonMapper
```

Comment:
It misses the description of the Javadoc parameter

Reply:
resolved via https://github.com/micronaut-projects/micronaut-core/pull/7117/commits/d0ab0629290e48d1d0cf1a5d06e571004085d41c